### PR TITLE
Clarify instructions to remove confusion

### PR DESCRIPTION
**Pull request by bhagerty**
I found the instructions for https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/accessing-object-properties-with-variables very confusing as written. The first sentence tells us to "Use the `playerNumber` variable to look up player 16 in `testObj` using bracket notation." This seems to suggest that setting the `playerNumber` variable in the first place somehow involves `testObj`. But it does not. The `playerNumber` variable is simply set to the number 16. It is only when `player` gets a value that we do anything with `testObj`. The point of this exercise is to show how the reader can assign a key (16) to a variable and then look up the key's associated value by using the variable rather than the key itself. I think that this comes across much more clearly, without confusion, in my proposed revised instruction.

Note: I can imagine someone saying "Well, we want the reader to figure it out, so it's okay if the instructions are confusing." I disagree. We shouldn't be challenging the reader to understand confusing instructions. In general, the FreeCodeCamp instructions are very clear. This problem is just an exception, and it is easy enough to make it clear without losing the fundamental point of the exercise.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/accessing-object-properties-with-variables.english.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Note that we do <em>not</em> use quotes around the variable name when using it t
 
 ## Instructions
 <section id='instructions'>
-Use the <code>playerNumber</code> variable to look up player <code>16</code> in <code>testObj</code> using bracket notation. Then assign that name to the <code>player</code> variable.
+Set the <code>playerNumber</code> variable to number <code>16</code>. You should then assign player 16's name to the <code>player</code> variable by looking up player 16's name in <code>testObj</code> with bracket notation and the <code>playerNumber</code> variable.
```
Comment (moT01) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/accessing-object-properties-with-variables.english.md:43]:
```suggestion
Set the <code>playerNumber</code> variable to <code>16</code>. Then assign player 16's name to the <code>player</code> variable by looking it up in <code>testObj</code> using bracket notation and the <code>playerNumber</code> variable.
```

I think this looks pretty good @bhagerty. I agree that this one is confusing without looking at the test text. I made a suggestion to try and make the instructions a little more concise. Perhaps I'm being too picky.

Comment (RandellDawson) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/accessing-object-properties-with-variables.english.md:43]: @bhagerty If you are fine with @moT01's, you can simply click the `Commit suggestion` button above.  I like Tom's suggestion and I believe it accomplishes what you were trying to do in the first place.

**Comment by moT01** [comment]
In the interest of saving time, I have committed the suggestion.

**Review by RandellDawson — APPROVED** [review]
LGTM

**Review by ShaunSHamilton — APPROVED** [review]
Hi @bhagerty . Congrats on your first pull request (PR)! 🎉

Thank you, for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**